src/test/resources/graphql-examples/deposit/addDeposit/addNewDepositNoBagName.graphql

```diff
@@ -1,5 +1,5 @@
 mutation {
-    addDeposit(input: {clientMutationId: "my-mutation-id", depositId: "00000000-0000-0000-0000-000000000006", creationTimestamp: "2019-05-15T23:36:00.000Z", depositorId: "archie001"}) {
+    addDeposit(input: {clientMutationId: "my-mutation-id", depositId: "00000000-0000-0000-0000-000000000006", creationTimestamp: "2019-05-15T23:36:00.000Z", depositorId: "archie001", origin: SMD}) {
```

Comment:
Please also include the `origin` field in the following example queries:

* `deposit/findDeposit/plain.graphql`
* `deposits/listAllDeposits/*.graphql`